ibis-server/app/model/metadata/trino.py

```diff
@@ -0,0 +1,135 @@
+from json import loads
+from urllib.parse import urlparse
+
+import pandas as pd
+
+from app.model import ConnectionUrl, TrinoConnectionInfo
+from app.model.data_source import DataSource
+from app.model.metadata.dto import (
+    Column,
+    Constraint,
+    Table,
+    TableProperties,
+    WrenEngineColumnType,
+)
+from app.model.metadata.metadata import Metadata
+
+
+class TrinoMetadata(Metadata):
+    def __init__(self, connection_info: TrinoConnectionInfo | ConnectionUrl):
+        super().__init__(connection_info)
+
+    def get_table_list(self) -> list[Table]:
+        sql = """SELECT
+                t.table_catalog,
+                t.table_schema,
+                t.table_name,
+                c.column_name,
+                c.data_type,
+                c.is_nullable
+              FROM
+                tables t
+              JOIN
+                columns c 
```

Comment:
```suggestion
                information_schema.tables t
              JOIN
                information_schema.columns c 
```
Why not specify the schema in the SQL so that you don't have to change the default schema in L41 & L43